### PR TITLE
Personal/xiatia/generate UUID v2

### PR DESCRIPTION
**Pull request by ACMoretxj**
## Description

Added "generateUUIDV2" helper function.

## Testing

See "handlebars-helper.spec.js".

### CODE REVIEW

File: src/e2e-test/regression-test/util/interceptors.js [hunk 1]
```diff
@@ -1,72 +1,96 @@
-// -------------------------------------------------------------------------------------------------
```
Comment (moria97) [inline src/e2e-test/regression-test/util/interceptors.js:1]: Can you check the line endings in these files?

Comment (ACMoretxj) [reply src/e2e-test/regression-test/util/interceptors.js:1]: Yes and I'm just checking it now. "git config --global core.autocrlf true" it set many times, may be history issues.

File: src/lib/handlebars-converter/handlebars-helpers.js [hunk 2]
```diff
@@ -735,6 +735,14 @@ module.exports.external = [
             return uuidv3(''.concat(urlNamespace), uuidv3.URL);
         }
     },
+    {
+        name: 'generateUUIDV2',
+        description: 'Generates a guid based on a URL: generateUUID url, Keep the results consistent across platforms, regardless of the platform\'s newline characters',
```
Comment (moria97) [inline src/lib/handlebars-converter/handlebars-helpers.js:740]: GUID

**Review by moria97 — DISMISSED** [review]

File: src/lib/handlebars-converter/handlebars-helpers.js (hunk 2 above)
Comment (ACMoretxj) [reply src/lib/handlebars-converter/handlebars-helpers.js:740]: sure, I will fix it.

**Review by moria97 — DISMISSED** [review]

**Review by moria97 — APPROVED** [review]

**Pull request merged**